program-test/tests/sysvar.rs

```diff
@@ -24,6 +24,10 @@ fn sysvar_getter_process_instruction(
     let rent = Rent::get()?;
     assert_eq!(rent, Rent::default());
 
+    // epoch rewards sysvar should not exist for banks that are not in reward period
+    let epoch_rewards = EpochRewards::get();
+    assert!(epoch_rewards.is_err());
```

Comment:
I worry that this test is a bit brittle, since there's nothing in the test that explicitly sets it up to be out of the reward period. Your other test checks both cases explicitly in `epoch_reward_sysvar_getter_process_instruction`, so I don't see the need for the check here. Please correct me if I'm missing something though!